queries.sql

```diff
@@ -75,4 +75,17 @@ INNER JOIN species ON animals.species_id = species.id WHERE vets.name = 'Maisy S
 insert into vets(name, age, date_of_graduation) values('William Tatcher', 45, '2000-04-23');
 insert into vets(name, age, date_of_graduation) values('Maisy Smith', 26, '2019-01-17');
 insert into vets(name, age, date_of_graduation) values('Stephanie Mendez', 64, '1981-05-04');
-insert into vets(name, age, date_of_graduation) values('Jack Harkness', 38, '2008-06-08');
\ No newline at end of file
+insert into vets(name, age, date_of_graduation) values('Jack Harkness', 38, '2008-06-08');
+
+
+-- performance tuning 
+
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
```

Comment:
@1GORDON 
Thanks for your review 
Please clarify your point as my initial execution time is > 1000ms based on the below requirement.

4 Depening on your machine speed, it might be enough or not. Check that by running explain analyze SELECT COUNT(*) FROM visits where animal_id = 4: - **_If you get Execution time: X ms and X >= 1000: that should be enough,_** you can continue to the project requirements. - If you get Execution time: X ms and X < 1000: please go back to point 3. and repeat until you get a value bigger than 1000ms.

Reply:
Hello team ✋.

- [ ] You are good to go if you have achieved an initial execution time below 1000s. 
- [ ] You may also kindly update screenshots as they might mislead the next reviewer.